### DISCUSSION TITLE
R query - issue with running table names - fully-qualified vs scoped queries?

### DISCUSSION THREAD

**Discussion by lgraham333** [redivis/meta#185] [Q&A] [answered]
According to the Redivis documentation's [(linked here)](https://apidocs.redivis.com/client-libraries/redivis-r/examples) second example for R code states in the code comments 

"We don't need to include fully-qualified table names if we scope our query to the appropriate dataset or project". 

When I run the given example code in R ...
`data <- (
    redivis::organization("Demo")
    $dataset("CMS 2014 Medicare Data")
    $query("
        SELECT provider_name, average_total_payments 
        FROM nursing_facilities
        INNER JOIN outpatient_charges USING (provider_id)
        WHERE state = 'CA'
    ")
    $to_tibble() 
)`

 I get the error ...

`Error in make_request(method = "POST", path = "/queries", payload = payload) : 
  Cannot look up unqualified table (nursing_facilities). Must be of the form "owner_name.project|dataset_name.table_name". Or perhaps you meant to set a default project?`

Can someone help explain what is going on? (The other examples work from this webpage when I run them in R.) 

Thank you!

**Comment by imathews** [comment, marked as answer]
Hi @lgraham333 — thanks for opening this! At first look, this appears to be a bug on our end. I just opened an issue in the `redivis-r` repo (https://github.com/redivis/redivis-r/issues/4), and will try to get it sorted asap
